ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -661,9 +661,9 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
                                                              bool* result) {
     // TODO: replace current_backend ->
     // BackendManager::GetCurrentlySetBackendName()
-    auto config_map =
-        BackendManager::GetBackendAttributeValues(current_backend);
-    *result = (config_map.at("ngraph_backend") == "NNPI");
+    // FIXME(gopoka): GatherV2 is supported only when axis == 0.
+    //                If possible, check should be added here.
```

Comment:
No, a check cant be added here. Here we can only check for "attributes" of the node. But the tf_axis check is an input to the node, which will be known only at runtime.

Reply:
Ok, I assumed that because `axis` is a static input it could be possibly checked here.

Is there any other way to remove node from being clustered when this kind of constraint is not satisfied?

Reply:
We can only check `attributes` here. The graph is not running yet, so we do not have access to values feeding into that node yet. So its fine to accept the node here, and later fail in `TranslateGraph`

Reply:
@gopoka  this comment is obsolete, is it?. please remove
```
// FIXME(gopoka):  GatherV2 is supported only when axis == 0.
    //                If possible, check should be added here.
```

Reply:
Removed.